docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -169,13 +158,14 @@ Let us consider the following example that defines a template for a mail sending
 
 The example defines five custom fields, each mapped to different technical properties:
 
-- _Implementation Type_ is mapped to the `camunda:class` property in BPMN 2.0 XML
-- _Sender_, _Receivers_ and _Template_ properties are mapped to `input parameters`
-- _Result Status_ is mapped back from the Java Delegate into a process variable via an `output parameter`
+- The task type `http` is mapped to the `zeebe:taskDefinition:type` property in BPMN 2.0 XML
+- The `REST Endpoint URL` and `REST Method` are mapped to `task headers`
+- The `Request Body` is mapped to a local variable via an `input parameter`
+- The `Result Variable` is mapped into a process variable via an `output parameter`
 
-All but the _Implementation Type_ are editable by the user through the properties panel as shown in the following screenshot:
+The task type is hidden to the user, and other properties are editable by the user through the properties panel as shown in the following screenshot:
```

Comment:
This sentence is slightly confusing. The fact of the matter is that _only_ the properties specified as part of the template are editable. No other properties are editable by default.

Reply:
How about:

```suggestion
The task type is hidden to the user, and other properties specified in the template are editable by the user through the properties panel as shown in the following screenshot:
```

Reply:
```suggestion
The task type is hidden to the user. Properties specified in the template can be edited through the properties panel as shown in the following screenshot:
```